engine/source/game/shapeBase.cpp

```diff
@@ -2874,7 +2874,7 @@ void ShapeBase::queueCollision(ShapeBase* obj, const VectorF& vec)
 
     while (ptr) {
         if (ptr->objectNumber == num) {
-            if (ptr->expireTime < time) {
+            if (ptr->expireTime < time || (obj->getTypeMask() & ItemObjectType) != 0) {
```

Comment:
Thanks for researching this!

Curious, is the `!= 0` needed for a more accurate decompile?

Reply:
I only put that to stay consistent with the code style.